ppublish.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Just curious, did you write this script yourself?

- If not: would it be possible to download it on-demand?
- If yes: what is your reasoning for checking if `.gitignore` exists and erroring if it does not?

Reply:
I have written the script myself. If I upload it somewhere, how we download it then?

This script exists because we want to put `dist` in gitignore. If gitignore does not exist, this script is pointless.

Reply:
>  have written the script myself. If I upload it somewhere, how we download it then?

Then it's not necessary. If you're still interested we could upload it as a gist and use either something like curl, but preferably an NPM package for simplicity, to download it.

> This script exists because we want to put dist in gitignore. If gitignore does not exist, this script is pointless.

I would say it's fair to assume we have a `.gitignore` in this project, but I guess it doesn't hurt to check.

Reply:
OK. I can make this an npm package. When I made one, I will create another PR to replace this.